src/LocalDate.js

```diff
@@ -1496,9 +1496,9 @@ export class LocalDate extends ChronoLocalDate{
      * @return {number} the Epoch Day equivalent to this date
      */
     toEpochDay() {
-        var y = this.year();
-        var m = this.monthValue();
-        var total = 0;
+        const y = this.year();
+        const m = this.monthValue();
```

Comment:
could be replaced with access to the "private" field instead of the function call?